### DISCUSSION TITLE
no modifications take effect

### DISCUSSION THREAD

**Discussion by jaymzh** [pqrs-org/Karabiner-Elements#3638] [General]
I had originally thought I was having [an issue with just complex modifications](https://github.com/pqrs-org/KE-complex_modifications/discussions/1532). But I decided to try a simple one since the event-viewer would help me understand what was going on.

I made a simple a -> b modification:

<img width="521" alt="image" src="https://github.com/pqrs-org/Karabiner-Elements/assets/938467/8cdc6c4a-a831-4941-bdbe-dab265d248f2">

But it does not work. The event viewer shows only `a` when I hit `a`.

I believe I've given KE all relevant permissions: `_grabber` and `_observer` both have `Input Monitoring` and I clicked `Allow` for the VirtualHID device. Under login items', KE is enabled, but the VHID entry isn't in the list.

**Comment by weierz** [comment]
Did you enable the **Modify events** for that keyboard in **Devices**?

<img width="1101" alt="image" src="https://github.com/pqrs-org/Karabiner-Elements/assets/14329786/5094b376-97b6-490a-924c-9d2aba4845d8">

**Comment by jaymzh** [comment]
Oh I had missed that - thanks!
